python/lsst/daf/butler/cli/util/split_commas.py

```diff
@@ -0,0 +1,49 @@
+# This file is part of daf_butler.
```

Comment:
For utility functions we don't require a separate file per function. The reason I wanted separate files per sub command was that I was worried we were going to get many sub commands and things would get out of control. We envisaged that there would be support code needed for the subcommands and that would get confusing. You seem to have mitigated a lot of that worry with clever callbacks in click definitions but we'll see what happens when more complex commands turn up.

Reply:
I'll move utils into their own file. I think it's worth keeping commands and options in separate files at least for now.